api/src/main/java/org/openmrs/module/kenyaemr/calculation/library/hiv/NeedsCACXTestCalculation.java

```diff
@@ -110,7 +108,22 @@ public CalculationResultMap evaluate(Collection<Integer> cohort, Map<String, Obj
                     false
             );
 
-            Encounter lastCacxScreeningEnc = EmrUtils.lastEncounter(patient, cacxEncType, cacxScreeningForm);
+            Encounter lastCacxScreening = EmrUtils.lastEncounter(patient, cacxEncType, cacxScreeningForm);
```

Comment:
I think we have another method that takes a list of encounter types and get the last. That could be a better fit here. Please check it out

Reply:
Checking it out asap